client/client_test.go

```diff
@@ -0,0 +1,44 @@
+package client_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/rddl-network/shamir-shareholder-service/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetMnemonic(t *testing.T) {
+	t.Parallel()
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, http.MethodGet, r.Method)
+		assert.Equal(t, "/mnemonic", r.URL.Path)
+		w.WriteHeader(http.StatusOK)
+		_, err := w.Write([]byte(`{"mnemonic":"word1"}`))
```

Comment:
only a suggestion: maybe create a real mnemonic. this just doesn't look right :)